src/internal/svg/fundWallet.tsx

```diff
@@ -0,0 +1,19 @@
+import { fill } from '../../styles/theme';
+
+export const FundWalletSvg = (
+  <svg
+    role="img"
+    aria-label="fund-wallet-svg"
+    width="16"
+    height="16"
+    viewBox="0 0 16 16"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      d="M8 16C12.4183 16 16 12.4183 16 8C16 3.58172 12.4183 0 8 0C3.58172 0 0 3.58172 0 8C0 12.4183 3.58172 16 8 16ZM1.63962 4.33685C5.1477 6.36224 9.63276 5.16047 11.6582 1.65239C9.63276 5.16047 10.8345 9.64553 14.3413 11.6702C10.8345 9.64553 6.35021 10.846 4.32482 14.3541C6.35021 10.846 5.1477 6.36224 1.63962 4.33685Z"
+      fill="#0A0B0D"
```

Comment:
Does this support the Dark Theme style? cc @kyhyco

Reply:
<img width="382" alt="image" src="https://github.com/user-attachments/assets/5f50778e-79b3-423e-a78b-adb3537d477c">

<img width="382" alt="image" src="https://github.com/user-attachments/assets/b9642059-0e9d-4873-970c-8f02eaa72cca">